scipy/spatial/spherical_voronoi.py

```diff
@@ -0,0 +1,342 @@
+"""
+Spherical Voronoi Code
+
+.. versionadded:: 0.17.0
+
+"""
+#
+# Copyright (C)  Tyler Reddy, Ross Hemsley, Edd Edmondson,
+#                Nikolai Nowaczyk, Joe Pitt-Francis, 2015.
+#
+# Distributed under the same BSD license as Scipy.
+#
+
+import numpy as np
+import numpy.matlib
+import scipy
+import math
+
+__all__ = ['SphericalVoronoi']
+
+
+def determinant_fallback(m):
```

Comment:
Can this function be simply replaced by `np.linalg.det`?
"Compatiblity with Python 2.6" is probably not the reason why it exists; whether np.linalg.det works or not shouldn't depend on that.